### ISSUE TITLE
Add a role to the group

### ISSUE THREAD

**Issue by justinehell** [ditrit/leto-modelizer-admin#78]
Blocked by #84 

On GroupPage, create the button to "Add a role" (above the second table)

![image](https://github.com/ditrit/leto-modelizer-admin/assets/62500081/f73307e1-5be8-4d04-b08e-7bb66c1d4149)

**Comment by justinehell** [comment]
Task put on hold, will be done when the new api is up.
